t/copy_input_fastq.t

```diff
@@ -0,0 +1,27 @@
+#!/bin/usr/env perl
+use warnings;
+use strict;
+
+use lib '/Warehouse/Users/csieffert/core-phylogenomics/lib';
+use lib '/Warehouse/Users/csieffert/core-phylogenomics/lib/Stage';
+use Test::More;
+use Test::Exception;
+use Stage::CopyInputFastq;
+use JobProperties;
+use Logger;
+
+#tests that CopyInputFastq.pm is correctly verifying that the input fastq and reference fasta have unique basenames.
+
+my $logger = Logger->new('.');
+my $properties = JobProperties->new('/Warehouse/Users/csieffert/core-phylogenomics/t');
```

Comment:
Similar comment to above about absolute paths, use $FindBin::Bin to get the path to the `t/` directory.